addons/website/static/src/scss/website.edit_mode.scss

```diff
@@ -45,12 +45,14 @@ $-editor-messages-margin-x: 2%;
         opacity: 0.3;
     }
 
-    &.oe_structure.oe_empty, &[data-oe-type=html], .oe_structure.oe_empty {
-        &:empty {
+    &.oe_structure.oe_empty, .oe_structure.oe_empty {
+        /*Making the drop zone displayed on drag small*/
+        &.oe_structure_small > .oe_drop_zone.oe_insert:not(.oe_vertical):only-child {
```

Comment:
Why is there still this `oe_structure_small` addition? Did the specs change again?

Reply:
It's the class that make the zone only appears on drag. But i realize that my comment is wrong and the naming isn't clear

Reply:
> It's the class that make the zone only appears on drag.
But the spec is that we want that everywhere... what do we need to add a class to do it?